internal/common/enums.rs

```diff
@@ -245,6 +245,26 @@ macro_rules! for_each_enums {
                 /// A key on a keyboard was released.
                 KeyReleased,
             }
+
+            /// The role for the `accessible-role` property
+            enum AccessibleRole {
+                /// An item with role `none` is not accessible
+                none,
+                /// A role for anything that is a Button or behaves like one.
+                button,
+                /// A role for anything that is a CheckBox or behaves like one.
+                checkbox,
+                /// A role for anything that is a ComboBox or behaves like one.
+                combobox,
+                /// A role for anything that is a Slider or behaves like one.
+                slider,
+                /// A role for anything that is a SpinBox or behaves like one.
+                spinbox,
+                /// A role for anything that is a Tab or behaves like one.
+                tab,
+                /// A role for static Text items.
+                text,
```

Comment:
This role is current not used. Should it be applied to `Text` items by default? Should our examples apply it? Or should it be removed (since unused)?

Reply:
It *is* used: it is applied to the `Text` item by default. (see lower_accessibility.rs:69)

Reply:
Thanks! I missed that.